### DISCUSSION TITLE
[blocks]: Frame Motion Effects

### DISCUSSION THREAD

**Discussion by sbeckerdev** [shadcn-ui/ui#6018] [Blocks Request]
### Description

I would love to have effects in all components like frame motion integrated something like Next UI.

### Example

https://nextui.pro/

**Comment by anishshobithps** [comment]
@sbeckerdev Have a look at https://github.com/birobirobiro/awesome-shadcn-ui there are many options to choose from and even based on framer motion. 

https://ui.aceternity.com/ is one of them.
